bus-mapping/src/evm/opcodes/sstore.rs

```diff
@@ -0,0 +1,176 @@
+        state: &mut CircuitInputStateRef,
+        geth_steps: &[GethExecStep],
+    ) -> Result<Vec<ExecStep>, Error> {
+        let geth_step = &geth_steps[0];
+        let mut exec_step = state.new_step(geth_step)?;
+
+        let _call_id = state.call()?.call_id;
+        let contract_addr = state.call()?.address;
+
+        state.push_op(
+            &mut exec_step,
+            RW::READ,
+            CallContextOp {
+                call_id: state.call()?.call_id,
+                field: CallContextField::TxId,
+                value: Word::from(state.tx_ctx.id()),
+            },
+        );
+        state.push_op(
+            &mut exec_step,
+            RW::READ,
+            CallContextOp {
+                call_id: state.call()?.call_id,
+                field: CallContextField::RwCounterEndOfReversion,
+                value: Word::from(state.call()?.rw_counter_end_of_reversion),
+            },
+        );
+        state.push_op(
+            &mut exec_step,
+            RW::READ,
+            CallContextOp {
+                call_id: state.call()?.call_id,
+                field: CallContextField::IsPersistent,
+                value: Word::from(state.call()?.is_persistent as u8),
+            },
+        );
+        state.push_op(
+            &mut exec_step,
+            RW::READ,
+            CallContextOp {
+                call_id: state.call()?.call_id,
+                field: CallContextField::CalleeAddress,
+                value: state.call()?.address.to_word(),
+            },
+        );
+
+        let key = geth_step.stack.nth_last(0)?;
+        let key_stack_position = geth_step.stack.nth_last_filled(0);
+        let value = geth_step.stack.nth_last(1)?;
+        let value_stack_position = geth_step.stack.nth_last_filled(1);
+
+        state.push_stack_op(&mut exec_step, RW::READ, key_stack_position, key)?;
+        state.push_stack_op(&mut exec_step, RW::READ, value_stack_position, value)?;
+
+        let is_warm = state
+            .sdb
+            .check_account_storage_in_access_list(&(contract_addr, key));
+
+        let (_, value_prev) = state.sdb.get_storage(&contract_addr, &key);
+        let value_prev = *value_prev;
+        let (_, committed_value) = state.sdb.get_committed_storage(&contract_addr, &key);
+        let committed_value = *committed_value;
+
+        state.push_op_reversible(
+            &mut exec_step,
+            RW::WRITE,
+            StorageOp::new(
+                state.call()?.address,
+                key,
+                value,
+                value_prev,
+                state.tx_ctx.id(),
+                committed_value,
+            ),
+        )?;
+
+        state.push_op_reversible(
+            &mut exec_step,
+            RW::WRITE,
+            TxAccessListAccountStorageOp {
+                tx_id: state.tx_ctx.id(),
+                address: state.call()?.address,
+                key,
+                value: true,
+                value_prev: is_warm,
```

Comment:
We should maybe consider to rename these two fields for something much closer to the actual values passed. `value` instead of true should contain a meaningful number/string IMO.

WDYT @han0110 @ed255